src/scss/style.scss

```diff
@@ -1,8 +1,8 @@
-@import 'partials/globals',
-        'partials/mixins',
-        'partials/responsive',
-        'partials/typography', 
-        'partials/variables';
+@use 'partials/globals' as *;
```

Comment:
import directives replaced with use - see relevant partials for other substitutions

Reply:
Reviewed and accepted.